hw/ip/otbn/doc/_index.md

```diff
@@ -596,6 +530,76 @@ When it executes this instruction, OTBN:
 The DMEM can be used to pass data back to the host processor, e.g. a "return value" or an "exit code".
 Refer to the section [Passing of data between the host CPU and OTBN]({{<relref "#writing-otbn-applications-datapassing" >}}) for more information.
 
+## Using hardware loops
+
+OTBN provides two hardware loop instructions, `LOOP` and `LOOPI`.
```

Comment:
Nit: If we're touching this, maybe turn this into a colon ("instructions: LOOP and LOOPI"). Otherwise it looks like it provides four things: two instructions, LOOP and LOOPI.